### DISCUSSION TITLE
Changes in config doesnt work

### DISCUSSION THREAD

**Discussion by Aisopia** [jagrosh/MusicBot#1372] [Questions]
tried to change the prefix in config but it doesn't work. pls help

**Comment by Lastorder-DC** [comment]
If you set server-specific prefix, prefix in setting won't work. Try to change server prefix using `!!prefix` command
(so If your current prefix is !, you can change using `!prefix #`)

If you never set server-specific prefix or above are not working, make sure that you changed right config file(besides with jar file) and reloaded bot after config change

**Comment by MattW0001** [comment]
I haven't been able to get this to work either, were you able to figure it out?

**Comment by MattW0001** [comment]
I just found the answer! https://github.com/jagrosh/MusicBot/issues/1215
